common.mk

```diff
@@ -194,55 +194,45 @@ else
 BUILDROOT_TOOLCHAIN=toolchain-aarch$(COMPILE_NS_USER)-legacy
 endif
 endif
-BUILDROOT_GETTY_PORT ?= \
-	$(if $(CFG_NW_CONSOLE_UART),ttyAMA$(CFG_NW_CONSOLE_UART),ttyAMA0)
+
+BR2_PACKAGE_LIBOPENSSL ?= y
+BR2_PACKAGE_MMC_UTILS ?= y
+BR2_PACKAGE_OPENSSL ?= y
+BR2_PACKAGE_OPTEE_BENCHMARK ?= $(CFG_TEE_BENCHMARK)
+BR2_PACKAGE_OPTEE_BENCHMARK_SITE ?= $(BENCHMARK_APP_PATH)
+BR2_PACKAGE_OPTEE_CLIENT_SITE ?= $(OPTEE_CLIENT_PATH)
+BR2_PACKAGE_OPTEE_EXAMPLES ?= y
+BR2_PACKAGE_OPTEE_EXAMPLES_CROSS_COMPILE ?= $(CROSS_COMPILE_S_USER)
+BR2_PACKAGE_OPTEE_EXAMPLES_SDK ?= $(OPTEE_OS_TA_DEV_KIT_DIR)
+BR2_PACKAGE_OPTEE_EXAMPLES_SITE ?= $(OPTEE_EXAMPLES_PATH)
+# The OPTEE_OS package builds nothing, it just installs files into the
+# root FS when applicable (for example: shared libraries)
+BR2_PACKAGE_OPTEE_OS ?= y
+BR2_PACKAGE_OPTEE_OS_SDK ?= $(OPTEE_OS_TA_DEV_KIT_DIR)
+BR2_PACKAGE_OPTEE_OS_SITE ?= $(CURDIR)/br-ext/package/optee_os
+BR2_PACKAGE_OPTEE_TEST ?= y
+BR2_PACKAGE_OPTEE_TEST_CROSS_COMPILE ?= $(CROSS_COMPILE_S_USER)
+BR2_PACKAGE_OPTEE_TEST_SDK ?= $(OPTEE_OS_TA_DEV_KIT_DIR)
+BR2_PACKAGE_OPTEE_TEST_SITE ?= $(OPTEE_TEST_PATH)
+BR2_PACKAGE_STRACE ?= y
+BR2_TARGET_GENERIC_GETTY_PORT ?= $(if $(CFG_NW_CONSOLE_UART),ttyAMA$(CFG_NW_CONSOLE_UART),ttyAMA0)
+
+# All BR2_* variables from the makefile or the environment are appended to
+# ../out-br/extra.conf unless already present
+# All values are quoted "..." except y and n
+double-quote = "#" # This really sets the variable to " and avoids upsetting vim's syntax highlighting
+streq = $(and $(findstring $(1),$(2)),$(findstring $(2),$(1)))
+y-or-n = $(or $(call streq,y,$(1)),$(call streq,n,$(1)))
+append-var_ = grep -q '^$(1)=' $(2) || echo '$(1)=$(3)'$($(1))'$(3)' >>$(2);
```

Comment:
Why is the grep needed?

Reply:
Yeah sorry about that I have removed it already. Initially I had not removed all the `@echo ...` lines and I thought they should have higher priority. Useless now.